components/node/eth/output.go

```diff
@@ -35,24 +35,28 @@ func (o *OutputResponse) ToOutputRootProof() bindings.TypesOutputRootProof {
 	}
 }
 
-func (o *OutputResponse) ToPublicInput(ChainId *big.Int) (bindings.TypesPublicInput, error) {
+func (o *OutputResponse) ToPublicInput() (bindings.TypesPublicInput, error) {
 	if o.NextBlock == nil {
 		return bindings.TypesPublicInput{}, ErrBlockIsEmpty
 	}
+	var withdrawalsRoot common.Hash
+	if o.NextBlock.WithdrawalsHash != nil {
+		withdrawalsRoot = *o.NextBlock.WithdrawalsHash
+	}
 	txHashes := make([][32]byte, len(o.NextTransactions))
 	for i, tx := range o.NextTransactions {
 		txHashes[i] = tx.Hash()
 	}
 	return bindings.TypesPublicInput{
-		Coinbase:         o.NextBlock.Coinbase,
+		BlockHash:        o.NextBlockRef.Hash,
+		ParentHash:       o.BlockRef.ParentHash,
```

Comment:
The `ParentHash` should be the current block hash, `o.BlockRef.Hash`.

Reply:
Sorry about that! I fixed it